lib/data_update_scripts/20201229230456_remove_collective_noun_from_site_config.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveCollectiveNounFromSiteConfig
+    def run
+      SiteConfig.where(var: %w[collective_noun collective_noun_disabled])&.destroy_all
```

Comment:
a small tip: & in this context is not needed. If no row is returned, then Rails will just not delete anything. There's no scenario where .where() can return nil. It will return an empty collection (precisely an "empty active record relation") if no row is found and thus .destroy_all will be a no-op executing nothing

Reply:
Thank you for sharing, @rhymes ! I've reverted this change. 😊